VolleyPaint/Assets/Scripts/Game/TeamAssignment.cs

```diff
@@ -15,8 +15,7 @@ void Start()
         assignedTeam = GameObject.Find("GameManager").GetComponent<GameManagement>().GetTeamToAutoAssignTo();
 
         // turn off character controller before teleporting
-        CharacterController cc = gameObject.GetComponent<CharacterController>();
```

Comment:
what is this change for?

Reply:
not sure if this is kims changes

Reply:
I think we should merge Kim's branch when he's ready. Keeping this change for now shouldn't cause a merge conflict, right?